### ISSUE TITLE
Screen reader does not announce the error info as ‘This field is required’ which appears after activating ‘Elementary button’

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#124]
User Experience:
If user can not perceive additional information is provided by screen reader then the people with disabilities and cognitive limitations will not be access the instructions for using the content.

Note: User credentials should NOT be included in the bug.

 

Repro-Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Left nav and select Pages button.

5. Navigate to the "Pages" window and select "Add page" button.

6. Enable screen reader navigate to the various controls of the "Add page" dialog like Title, Permalink, Description etc.

7. Verify the issue.

 

Actual Result:

Screen reader does not announce the error info as ‘This field is required’ which appears after activating ‘Elementary button’ error present in ‘Permalink’ input field.

 

Expected result:

Screen reader should announce the error info as ‘This field is required’ which appears after activating ‘Elementary button’ error ‘Permalink’ input field.

 

Note: 

Similar issue exist with Chrome.

Similar issue exist throughout the application

 

Reference Links

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
